### PR TITLE
[Case Study] BettyCrocker wake lock case study

### LINKED ISSUE #2267: content: Wake Lock API case study [Apr 25]
* Deadline: 2020 April 25 (hard)
* [Draft](https://docs.google.com/document/d/1BSN7mnGBTiEBk48LU9oeCtlYno9z33wKnos5x77x80o/edit?hl=en)
* Author: @tomayac 
* Reviewer: @jpmedley 

@tomayac if this API isn't supported in other browsers (I can't find MDN or Can I use? articles on the topic) please make sure to mention at the top of the article that this is only supported in Chrome. [Example](https://deploy-preview-2235--web-dev-staging.netlify.com/snap-after-layout/#interoperability).

### PR DESCRIPTION
**Pull request by tomayac**
Fixes #2267

Changes proposed in this pull request:

- Adds the case study.

### CODE REVIEW

File: src/site/content/en/blog/betty-crocker/index.md [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+Launched in 1997, [BettyCrocker.com](https://www.bettycrocker.com/)
+continues the legacy of adapting, discovering, and sharing the knowledge that home cooks crave.
+Today, the site receives more than 12&nbsp;million visitors per month.
+
+## The retired iOS and Android apps
+
+Launched to [much fanfare](https://consumergoods.com/betty-crocker-launches-cookbook-app) in 2014,
+Betty Crocker recently took their native apps out of the Apple App Store and the Google Play Store
+after it had been mostly neglected and deprioritized.
+For a long time, the Betty Crocker team have preferred adding features to the mobile site
```
Comment (philkrie) [inline src/site/content/en/blog/betty-crocker/index.md:36]: For a long time, the Betty Crocker team has preferred adding new features to the mobile site instead of the native apps

File: src/site/content/en/blog/betty-crocker/index.md [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+  […]—[@Katie_Tweedy_](https://twitter.com/Katie_Tweedy_/status/996746567762763776)
+
+> Shout to [@livefront](https://twitter.com/livefront) in minneapolis
+  who solved for the screen dim issues on [@BettyCrocker](https://twitter.com/BettyCrocker) app.
+  It takes a village. […]—[@aimeereker](https://twitter.com/aimeereker/status/996751679415570435)
+
+## The Wake Lock API
+
+When cooking with a device, there is nothing more frustrating
+than having to touch the screen with messy hands or even your nose when the screen locks up.
```
Comment (philkrie) [inline src/site/content/en/blog/betty-crocker/index.md:63]: locks up suggests it froze... "when the screen turns off"

File: src/site/content/en/blog/betty-crocker/index.md [hunk 3]
```diff
@@ -0,0 +1,352 @@
...
+Betty Crocker asked themselves how they could port the killer feature of their native app
+over to the web app.
+This is when they learned about
+[Project Fugu](https://developers.google.com/web/updates/capabilities) and the
+[Wake Lock API](https://web.dev/wakelock/)
+that is being developed in the context of the project.
+
```
Comment (philkrie) [inline src/site/content/en/blog/betty-crocker/index.md:70]: delete "that is being developed in the context of the project"

File: src/site/content/en/blog/betty-crocker/index.md [hunk 4]
```diff
@@ -0,0 +1,352 @@
...
+![Doughy fingers](doughy-fingers.jpg)
+
+The Wake Lock API provides a way to prevent the device
+from dimming and locking the screen or going to sleep.
```
Comment (philkrie) [inline src/site/content/en/blog/betty-crocker/index.md:74]: from dimming or locking the screen.

File: src/site/content/en/blog/betty-crocker/index.md [hunk 5]
```diff
@@ -0,0 +1,352 @@
...
+  versus 1min 38sec for users who did not.
+- The bounce rate for users who enabled the wake lock was 21.85%,
+  versus 56.51% for users who did not.
+- During a one month time period, 27,563 users enabled the wake lock multiple times.
+- After enabling the wake lock, 1.43% of users intended to purchase ingredients
+  (measured by coupon code clicks), versus 0.37% for all users.
+
+## Conclusions
+
+Betty Crocker have seen fantastic results with the Wake Lock API.
```
Comment (philkrie) [inline src/site/content/en/blog/betty-crocker/index.md:332]: Betty Crocker has seen fantastic results using the Wake Lock API.

File: src/site/content/en/blog/betty-crocker/index.md [hunk 6]
```diff
@@ -0,0 +1,352 @@
...
+        GeneralMills.Analytics && GeneralMills.Analytics.raiseEvent(eventParams);
+      },
+    },
+  };
+</script>
+```
+
+### Results
+
+The Vue.js component has been deployed on the three sites and delivered great results.
```
Comment (philkrie) [inline src/site/content/en/blog/betty-crocker/index.md:316]: The Vue.js component has been deployed on all three sites and delivered great results. During the period from December 10th, 2019 to January 10th, 2020, BettyCrocker.com reported the following metrics:

File: src/site/content/en/blog/betty-crocker/index.md [hunk 7]
```diff
@@ -0,0 +1,352 @@
...
+continues the legacy of adapting, discovering, and sharing the knowledge that home cooks crave.
+Today, the site receives more than 12&nbsp;million visitors per month.
+
+## The retired iOS and Android apps
+
+Launched to [much fanfare](https://consumergoods.com/betty-crocker-launches-cookbook-app) in 2014,
+Betty Crocker recently took their native apps out of the Apple App Store and the Google Play Store
+after it had been mostly neglected and deprioritized.
+For a long time, the Betty Crocker team have preferred adding features to the mobile site
+instead of to the native apps.
```
Comment (tomayac) [inline src/site/content/en/blog/betty-crocker/index.md:37]:
```suggestion
For a long time, the Betty Crocker team has preferred adding new features to the mobile site
instead of the native apps.
```

File: src/site/content/en/blog/betty-crocker/index.md [hunk 8]
```diff
@@ -0,0 +1,352 @@
...
+  […]—[@Katie_Tweedy_](https://twitter.com/Katie_Tweedy_/status/996746567762763776)
+
+> Shout to [@livefront](https://twitter.com/livefront) in minneapolis
+  who solved for the screen dim issues on [@BettyCrocker](https://twitter.com/BettyCrocker) app.
+  It takes a village. […]—[@aimeereker](https://twitter.com/aimeereker/status/996751679415570435)
+
+## The Wake Lock API
+
+When cooking with a device, there is nothing more frustrating
+than having to touch the screen with messy hands or even your nose when the screen locks up.
```
Comment (tomayac) [inline src/site/content/en/blog/betty-crocker/index.md:63]:
```suggestion
than having to touch the screen with messy hands or even your nose when the screen turns off.
```

File: src/site/content/en/blog/betty-crocker/index.md [hunk 9]
```diff
@@ -0,0 +1,352 @@
...
+## The Wake Lock API
+
+When cooking with a device, there is nothing more frustrating
+than having to touch the screen with messy hands or even your nose when the screen turns off.
+Betty Crocker asked themselves how they could port the killer feature of their native app
+over to the web app.
+This is when they learned about
+[Project Fugu](https://developers.google.com/web/updates/capabilities) and the
+[Wake Lock API](https://web.dev/wakelock/)
+that is being developed in the context of the project.
```
Comment (tomayac) [inline src/site/content/en/blog/betty-crocker/index.md:69]:
```suggestion
[Wake Lock API](https://web.dev/wakelock/).
```

File: src/site/content/en/blog/betty-crocker/index.md [hunk 10]
```diff
@@ -0,0 +1,351 @@
...
+Betty Crocker asked themselves how they could port the killer feature of their native app
+over to the web app.
+This is when they learned about
+[Project Fugu](https://developers.google.com/web/updates/capabilities) and the
+[Wake Lock API](https://web.dev/wakelock/).
+
+![Doughy fingers](doughy-fingers.jpg)
+
+The Wake Lock API provides a way to prevent the device
+from dimming and locking the screen or going to sleep.
```
Comment (tomayac) [inline src/site/content/en/blog/betty-crocker/index.md:73]:
```suggestion
from dimming or locking the screen.
```

File: src/site/content/en/blog/betty-crocker/index.md [hunk 11]
```diff
@@ -0,0 +1,351 @@
...
+    },
+  };
+</script>
+```
+
+### Results
+
+The Vue.js component has been deployed on the three sites and delivered great results.
+The measuring period lasted from December 10th, 2019 to January 10th, 2020
+and covered the site BettyCrocker.com.
```
Comment (tomayac) [inline src/site/content/en/blog/betty-crocker/index.md:317]:
```suggestion
The Vue.js component has been deployed on all three sites and delivered great results.
During the period from December 10th, 2019 to January 10th, 2020,
BettyCrocker.com reported the following metrics:
```

File: src/site/content/en/blog/betty-crocker/index.md [hunk 12]
```diff
@@ -0,0 +1,351 @@
...
+  versus 1min 38sec for users who did not.
+- The bounce rate for users who enabled the wake lock was 21.85%,
+  versus 56.51% for users who did not.
+- During a one month time period, 27,563 users enabled the wake lock multiple times.
+- After enabling the wake lock, 1.43% of users intended to purchase ingredients
+  (measured by coupon code clicks), versus 0.37% for all users.
+
+## Conclusions
+
+Betty Crocker have seen fantastic results with the Wake Lock API.
```
Comment (tomayac) [inline src/site/content/en/blog/betty-crocker/index.md:331]:
```suggestion
Betty Crocker has seen fantastic results using the Wake Lock API.
```

**Comment by tomayac** [comment]
Thanks, @philkrie, I've addressed your comments. Pro-tip: use the [suggestion](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request:~:text=Optionally%2C%20to%20suggest%20a%20specific%20change,the%20text%20within%20the%20suggestion%20block.) feature the next time.

File: src/site/content/en/blog/betty-crocker/index.md [hunk 13]
```diff
@@ -0,0 +1,351 @@
...
+
+- Of all Betty Crocker users with a browser compatible with the Wake Lock API,
+  3.5% of them enabled the feature (~127,000 users).
+- The session duration for users who enabled the wake lock was 4min 18sec,
+  versus 1min 38sec for users who did not.
+- The bounce rate for users who enabled the wake lock was 21.85%,
+  versus 56.51% for users who did not.
+- During a one month time period, 27,563 users enabled the wake lock multiple times.
+- After enabling the wake lock, 1.43% of users intended to purchase ingredients
+  (measured by coupon code clicks), versus 0.37% for all users.
```
Comment (tomayac) [inline src/site/content/en/blog/betty-crocker/index.md:327]:
```suggestion
- Of all Betty Crocker users with a browser compatible with the Wake Lock API,
  3.5% of them enabled the feature immediately, making it a top-5 action.
- The session duration for users who enabled the wake lock was 3.1× longer
  than for users who did not.
- The bounce rate for users who enabled the wake lock was 50% lower
  for those using the wake lock feature.
- Coupon clicks (one indicator of purchase intent) was about 300% higher
  for wake lock users compared to all users.
```

**Comment by tomayac** [comment]
Staging preview deep link: https://deploy-preview-2494--web-dev-staging.netlify.com/betty-crocker/.

**Comment by kaycebasques** [comment]
@tomayac this is listed as a draft so I'm assuming that you're still working on it with others. Please ping me in a comment and assign this PR to me when you're ready for my review.

**Comment by tomayac** [comment]
@kaycebasques Correct, we're still waiting for the customer's marketing department to approve the current draft, and will then hand over to our team-internal reviewers (I guess: you).

**Comment by tomayac** [comment]
@kaycebasques I've tightened up the intro a bit, please take a look. Thank you!

File: src/site/content/en/blog/betty-crocker/index.md [hunk 14]
```diff
@@ -0,0 +1,340 @@
...
+today receives more than 12&nbsp;million visitors per month.
+After they **implemented the Wake Lock API**, their
+**indicators of purchase intent were about 300% higher**
+for wake lock users compared to all users.
+
+## The retired iOS and Android apps
+
+Released to [much fanfare](https://consumergoods.com/betty-crocker-launches-cookbook-app) in 2014,
+Betty Crocker recently took their native apps out of the Apple App Store and the Google Play Store
+after it had been mostly neglected and deprioritized.
```
Comment (kaycebasques) [inline src/site/content/en/blog/betty-crocker/index.md:35]:
```suggestion
after they had been mostly neglected and deprioritized.
```

This should be `they` because it's a reference to `native apps` (plural)

File: src/site/content/en/blog/betty-crocker/index.md [hunk 15]
```diff
@@ -0,0 +1,340 @@
...
+For a long time, the Betty Crocker team has preferred adding new features to the mobile site
+instead of the native apps.
+The technical platform the native apps were created on was outdated,
+and the business did not have the resources
+to support updating and maintaining the apps moving forward.
+In addition to that, the web app also was objectively way bigger traffic-wise,
```
Comment (kaycebasques) [inline src/site/content/en/blog/betty-crocker/index.md:41]:
```suggestion
The web app also was objectively way bigger traffic-wise,
```
We can remove these words without changing the meaning of the sentence

(The less words in general, the more likely someone is to read through the entire document)

File: src/site/content/en/blog/betty-crocker/index.md [hunk 16]
```diff
@@ -0,0 +1,340 @@
...
+more modern, and easier to enhance.
+
+The native apps did have one *killer feature*, though, that its users loved:
```
Comment (kaycebasques) [inline src/site/content/en/blog/betty-crocker/index.md:44]:
```suggestion
The native apps did have one *killer feature*, though, that their users loved:
```

File: src/site/content/en/blog/betty-crocker/index.md [hunk 17]
```diff
@@ -0,0 +1,340 @@
...
+
+> Millennial cooking pro tip: the [@BettyCrocker](https://twitter.com/BettyCrocker) mobile app
+  doesn't dim or lock when you're following a recipe.
+  […]—[@AvaBeilke](https://twitter.com/AvaBeilke/status/996746473168670720)
```
Comment (kaycebasques) [inline src/site/content/en/blog/betty-crocker/index.md:48]:
```suggestion
  —[@AvaBeilke](https://twitter.com/AvaBeilke/status/996746473168670720)
```
I think you can avoid the ellipsis. I get that we're using it to signify that it's not a complete quote, but you usually only need to use it when you're eliminating words from the middle of a sentence

File: src/site/content/en/blog/betty-crocker/index.md [hunk 18]
```diff
@@ -0,0 +1,340 @@
...
+
+> 80% of people cook with a device in the kitchen, but screen dimming and locking is a problem.
+  What did [@BettyCrocker](https://twitter.com/BettyCrocker) do?
+  Updated their app to NOT dim when users are in a recipe.
+  […]—[@Katie_Tweedy_](https://twitter.com/Katie_Tweedy_/status/996746567762763776)
```
Comment (kaycebasques) [inline src/site/content/en/blog/betty-crocker/index.md:53]:
```suggestion
  —[@Katie_Tweedy_](https://twitter.com/Katie_Tweedy_/status/996746567762763776)
```

File: src/site/content/en/blog/betty-crocker/index.md [hunk 19]
```diff
@@ -0,0 +1,340 @@
...
+
+## The Wake Lock API
```
Comment (kaycebasques) [inline src/site/content/en/blog/betty-crocker/index.md:55]:
```suggestion
## Bringing the killer feature to the web with the Wake Lock API
```

File: src/site/content/en/blog/betty-crocker/index.md [hunk 20]
```diff
@@ -0,0 +1,340 @@
...
+
+When cooking with a device, there is nothing more frustrating
+than having to touch the screen with messy hands or even your nose when the screen turns off.
+Betty Crocker asked themselves how they could port the killer feature of their native app
+over to the web app.
+This is when they learned about
+[Project Fugu](https://developers.google.com/web/updates/capabilities) and the
+[Wake Lock API](https://web.dev/wakelock/).
+
+![Doughy fingers](doughy-fingers.jpg)
```
Comment (kaycebasques) [inline src/site/content/en/blog/betty-crocker/index.md:65]:
```suggestion
![A person kneading dough on a kitchen table covered in flour](doughy-fingers.jpg)
```
If I couldn't see the picture I'm not sure I would know what `doughy fingers` means... like fingers made out of dough?

File: src/site/content/en/blog/betty-crocker/index.md [hunk 21]
```diff
@@ -0,0 +1,340 @@
...
+
+The Wake Lock API provides a way to prevent the device
+from dimming or locking the screen.
+This capability enables new experiences that, until now, required a native app.
+The Wake Lock API reduces the need for hacky and potentially power-hungry workarounds.
+
+### Requesting a wake lock
+
+To request a wake lock, you need to call the `navigator.wakeLock.request()` method
+that returns a `WakeLockSentinel` object.
```
Comment (kaycebasques) [inline src/site/content/en/blog/betty-crocker/index.md:75]:
```suggestion
that returns a `WakeLockSentinel` object. You'll use this object as a
[sentinel value](https://en.wikipedia.org/wiki/Sentinel_value).
```
Our audience probably knows what a sentinel is but I don't think it hurts to provide a link for the minority that have never heard that idea before

File: src/site/content/en/blog/betty-crocker/index.md [hunk 22]
```diff
@@ -0,0 +1,340 @@
...
+  wakeLock = null;
+}, 5000);
+```
+
+## The implementation
+
+With the new web app, users should be enabled to easily navigate through a recipe,
+complete steps, and even walk away without the screen locking.
+To achieve this goal, the team first built a quick front-end prototype
+as a proof the concept and to gather UX input.
```
Comment (kaycebasques) [inline src/site/content/en/blog/betty-crocker/index.md:118]:
```suggestion
as a proof of concept and to gather UX input.
```

File: src/site/content/en/blog/betty-crocker/index.md [hunk 23]
```diff
@@ -0,0 +1,340 @@
...
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="www.tablespoon.com.png" alt="Tablespoon.com wake lock toggle" width="400">
+  <figcaption class="w-figcaption">Tablespoon.com wake lock toggle.</figcaption>
+</figure>
+
+While focusing on the [`ViewModel` layer of the MVVM pattern](https://012.vuejs.org/guide/)
+in developing the component based on the new site's modernized framework,
+they developed with interoperability in mind
+to enable functionality on legacy and new frameworks of the site.
```
Comment (kaycebasques) [inline src/site/content/en/blog/betty-crocker/index.md:147]: This paragraph was hard for me to read. Can you rephrase it? I suggest breaking it up into smaller sentences.

File: src/site/content/en/blog/betty-crocker/index.md [hunk 24]
```diff
@@ -0,0 +1,340 @@
...
+The Vue.js component has been deployed on all three sites and delivered great results.
+During the period from December 10th, 2019 to January 10th, 2020,
+BettyCrocker.com reported the following metrics:
+
+- Of all Betty Crocker users with a browser compatible with the Wake Lock API,
+  3.5% of them enabled the feature immediately, making it a top-5 action.
+- The session duration for users who enabled the wake lock was 3.1× longer
+  than for users who did not.
+- The bounce rate for users who enabled the wake lock was 50% lower
+  for those using the wake lock feature.
```
Comment (kaycebasques) [inline src/site/content/en/blog/betty-crocker/index.md:315]:
```suggestion
  than for those not using the wake lock feature.
```

File: src/site/content/en/blog/betty-crocker/index.md [hunk 25]
```diff
@@ -0,0 +1,340 @@
...
+  - fugu
+draft: true
+---
+For nearly a century, Betty Crocker has been America's source for modern cooking instruction
+and trusted recipe development.
+Launched in 1997, their site [BettyCrocker.com](https://www.bettycrocker.com/)
+today receives more than 12&nbsp;million visitors per month.
+After they **implemented the Wake Lock API**, their
+**indicators of purchase intent were about 300% higher**
+for wake lock users compared to all users.
```
[the remark this answers is not in the document]
Comment (kaycebasques) [reply src/site/content/en/blog/betty-crocker/index.md:29]: We have a nice [stat component](https://web.dev/handbook/web-dev-components/#stats) that you can use to bring extra attention to the stat.

File: src/site/content/en/blog/betty-crocker/index.md [hunk 26]
```diff
@@ -0,0 +1,340 @@
...
+  for those using the wake lock feature.
+- Indicators of purchase intent were about 300% higher for wake lock users compared to all users.
+
+## Conclusions
+
+Betty Crocker has seen fantastic results using the Wake Lock API.
+You can test the feature yourself by searching for your favorite recipe on any of their sites
+([BettyCrocker](https://www.bettycrocker.com/),
+[Pillsbury](https://www.pillsbury.com/), or [Tablespoon](https://www.tablespoon.com/))
+and toggling the "Prevent your screen from going dark while you cook" toggle.
```
Comment (kaycebasques) [inline src/site/content/en/blog/betty-crocker/index.md:324]:
```suggestion
and enabling the **Prevent your screen from going dark while you cook** toggle.
```

Switching to `enabling` avoids having to use the word `toggle` redundantly

Style guide convention is to format UI labels in bold: https://developers.google.com/style/ui-elements#formatting

File: src/site/content/en/blog/betty-crocker/index.md [hunk 27]
```diff
@@ -0,0 +1,340 @@
...
+
+Use cases for wake locks do not stop at recipe sites.
+Other examples are boarding pass or ticket apps that need to keep the screen on
+until the barcode has been scanned, kiosk-style apps that keep the screen on continuously,
+or web-based presentation apps that prevent the screen from sleeping during a presentation.
+
+We have compiled [everything you need to know about the Wake Lock API](https://web.dev/wakelock/)
+in a comprehensive article over on our developer portal.
```
Comment (kaycebasques) [inline src/site/content/en/blog/betty-crocker/index.md:332]: The usage of `developer portal` seems funny because it's a link to the same site (web.dev)

File: src/site/content/en/blog/betty-crocker/index.md (hunk 20 above)
Comment (kaycebasques) [reply src/site/content/en/blog/betty-crocker/index.md:65]: Please run this one through an optimizer also if you haven't already

File: src/site/content/en/blog/betty-crocker/index.md [hunk 28]
```diff
@@ -0,0 +1,340 @@
...
+title: |
+  Better cooking with the Wake Lock API : A&nbsp;case study with Betty Crocker
+subhead: |
+  After they have implemented the Wake Lock API, cooking site BettyCrocker.com's
+  indicators of purchase intent were about 300% higher for wake lock users compared to all users.
+authors:
+  - thomassteiner
+date: 2020-04-22
+updated: 2020-04-22
+hero: hero.jpg
```
[the remark this answers is not in the document]
Comment (tomayac) [reply src/site/content/en/blog/betty-crocker/index.md:11]: This is a placeholder, we're waiting for their marketing department to provide us with a better one. Sorry, this could've been mentioned somewhere.

File: src/site/content/en/blog/betty-crocker/index.md (hunk 27 above)
Comment (tomayac) [reply src/site/content/en/blog/betty-crocker/index.md:332]:
```suggestion
in a comprehensive article on this very site.
```

True. I will just remove this part.

File: src/site/content/en/blog/betty-crocker/index.md [hunk 29]
```diff
@@ -0,0 +1,340 @@
...
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="www.tablespoon.com.png" alt="Tablespoon.com wake lock toggle" width="400">
+  <figcaption class="w-figcaption">Tablespoon.com wake lock toggle.</figcaption>
+</figure>
+
+While focusing on the [`ViewModel` layer of the MVVM pattern](https://012.vuejs.org/guide/)
+in developing the component based on the new site's modernized framework,
+they developed with interoperability in mind
+to enable functionality on legacy and new frameworks of the site.
```
Comment (tomayac) [inline src/site/content/en/blog/betty-crocker/index.md:147]:
```suggestion
When developing the component based on the new site's modernized framework,
there was a strong focus on the [`ViewModel` layer of the MVVM pattern](https://012.vuejs.org/guide/).
The team also developed with interoperability in mind
to enable functionality on legacy and new frameworks of the site.
```

File: src/site/content/en/blog/betty-crocker/index.md (hunk 23 above)
Comment (tomayac) [reply src/site/content/en/blog/betty-crocker/index.md:148]: Broken down.

File: src/site/content/en/blog/betty-crocker/index.md [hunk 30]
```diff
@@ -0,0 +1,341 @@
+---
+title: |
+  Wake Lock API case study: 300% increase in purchase intent indicators on BettyCrocker.com
+subhead: |
+  After they have implemented the Wake Lock API, cooking site BettyCrocker.com's
+  indicators of purchase intent were about 300% higher for wake lock users compared to all users.
```
Comment (tomayac) [inline src/site/content/en/blog/betty-crocker/index.md:6]:
```suggestion
  Nothing is worse when cooking with a mobile device than its screen turning off in the middle of a recipe step.
  Learn how cooking site BettyCrocker.com used the Wake Lock API for profit to prevent this from happening.
```

File: src/site/content/en/blog/betty-crocker/index.md (hunk 25 above)
[the remark this answers is not in the document]
Comment (tomayac) [reply src/site/content/en/blog/betty-crocker/index.md]: > We have a nice stat component that you can use to bring extra attention to the stat.

Now [using it](https://github.com/GoogleChrome/web.dev/pull/2494/commits/d1fb748f9316711337a016cd8982f2be543053a9).

File: src/site/content/en/blog/betty-crocker/index.md [hunk 31]
```diff
@@ -0,0 +1,356 @@
+---
+title: |
+  Wake Lock API case study: 300% increase in purchase intent indicators on BettyCrocker.com
+subhead: |
+  Nothing is worse when cooking with a mobile device than its screen turning off in the middle of a recipe step.
+  Learn how cooking site BettyCrocker.com used the Wake Lock API for profit to prevent this from happening.
```
Comment (kaycebasques) [inline src/site/content/en/blog/betty-crocker/index.md:6]:
```suggestion
  Learn how cooking site BettyCrocker.com used the Wake Lock API to prevent this from happening.
```

File: src/site/content/en/blog/betty-crocker/index.md [hunk 32]
```diff
@@ -0,0 +1,356 @@
...
+alt: Betty Crocker Cake Mix—Coffee Cinnamon.
+description: |
+  Case study of cooking site BettyCrocker.com's experiences with implementing the Wake Lock API.
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - case-study
+  - wake-lock
+  - capabilities
+  - fugu
+draft: true
```
Comment (kaycebasques) [inline src/site/content/en/blog/betty-crocker/index.md:21]:
```suggestion
```
Just leaving a reminder to remove this before merging

**Comment by philkrie** [comment]
@kaycebasques BC asked for more time to review and get back to us, we'll try to follow up this week. Also @tomayac would it be possible for me to be included as an author/contributor for the case study?

**Comment by tomayac** [comment]
@kaycebasques Thanks for the review and the approval, as @philkrie wrote, we're waiting for the final marketing approval and hero image from Betty.

@philkrie I have added you as an author. Do a hard-refresh on https://deploy-preview-2494--web-dev-staging.netlify.app/betty-crocker/ and you should show up :-)

**Comment by kaycebasques** [comment]
The linter wants you to replace the double quotes in `contributors.js` with single quotes

File: src/site/_data/contributors.js [hunk 33]
```diff
@@ -1687,6 +1687,22 @@ const contributors = {
...
+    github: "philkrie",
+    description: {
+      en: "Phillip is a Specialist Engineer",
+    },
+    org: {
+      name: "Google",
+      unit: "Global Business",
+    },
+    country: "US",
+  },
```
Comment (kaycebasques) [inline src/site/_data/contributors.js:1705]:
```suggestion
  phillipkriegel: {
    name: {
      given: 'Phillip',
      family: 'Kriegel',
    },
    homepage: 'https://www.philkrie.me/',
    github: 'philkrie',
    description: {
      en: 'Phillip is a Specialist Engineer',
    },
    org: {
      name: 'Google',
      unit: 'Global Business',
    },
    country: 'US',
  },
```

File: src/site/content/en/blog/betty-crocker/index.md [hunk 34]
```diff
@@ -0,0 +1,358 @@
...
+  - thomassteiner
+  - phillipkriegel
+date: 2020-04-22
+updated: 2020-04-29
+hero: hero.jpg
+alt: Betty Crocker Cake Mix—Coffee Cinnamon.
+description: |
+  Case study of cooking site BettyCrocker.com's experiences with implementing the Wake Lock API.
+tags:
+  - post # post is a required tag for the article to show up in the blog.
```
Comment (kaycebasques) [inline src/site/content/en/blog/betty-crocker/index.md:18]:
```suggestion
  - blog
```

File: src/site/_data/contributors.js [hunk 35]
```diff
@@ -1687,6 +1687,22 @@ const contributors = {
     github: "una",
     twitter: "una",
   },
+  phillipkriegel: {
```
Comment (robdodson) [inline src/site/_data/contributors.js:1690]: need a 2x version of phillip's headshot.

File: src/site/content/en/blog/betty-crocker/index.md [hunk 36]
```diff
@@ -0,0 +1,358 @@
...
+so they were able to implement the component once, and deploy it everywhere,
+as shown in the screenshots below.
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="www.bettycrocker.com.png" alt="BettyCrocker.com wake lock toggle" width="400">
+  <figcaption class="w-figcaption">BettyCrocker.com wake lock toggle.</figcaption>
+</figure>
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="www.pillsbury.com.png" alt="Pillsbury.com wake lock toggle" width="400">
```
Comment (robdodson) [inline src/site/content/en/blog/betty-crocker/index.md:138]:
```suggestion
  <img class="w-screenshot" src="www.pillsbury.com.png" alt="Pillsbury.com wake lock toggle" width="600">
```

File: src/site/content/en/blog/betty-crocker/index.md [hunk 37]
```diff
@@ -0,0 +1,358 @@
...
+[Vue.js component](https://vuejs.org/v2/guide/components.html)
+that could be shared across all their brands ([BettyCrocker](https://www.bettycrocker.com/),
+[Pillsbury](https://www.pillsbury.com/), and [Tablespoon](https://www.tablespoon.com/)).
+Even though only Betty Crocker had a native app,
+the three sites do have a shared code base,
+so they were able to implement the component once, and deploy it everywhere,
+as shown in the screenshots below.
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="www.bettycrocker.com.png" alt="BettyCrocker.com wake lock toggle" width="400">
```
Comment (robdodson) [inline src/site/content/en/blog/betty-crocker/index.md:133]:
```suggestion
  <img class="w-screenshot" src="www.bettycrocker.com.png" alt="BettyCrocker.com wake lock toggle" width="600">
```

File: src/site/content/en/blog/betty-crocker/index.md [hunk 38]
```diff
@@ -0,0 +1,358 @@
...
+  <figcaption class="w-figcaption">BettyCrocker.com wake lock toggle.</figcaption>
+</figure>
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="www.pillsbury.com.png" alt="Pillsbury.com wake lock toggle" width="400">
+  <figcaption class="w-figcaption">Pillsbury.com wake lock toggle.</figcaption>
+</figure>
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="www.tablespoon.com.png" alt="Tablespoon.com wake lock toggle" width="400">
```
Comment (robdodson) [inline src/site/content/en/blog/betty-crocker/index.md:143]:
```suggestion
  <img class="w-screenshot" src="www.tablespoon.com.png" alt="Tablespoon.com wake lock toggle" width="600">
```
